massa-execution-worker/src/tests/scenarios_mandatories.rs

```diff
@@ -807,7 +825,7 @@ fn datastore_manipulations() {
     );
 
     // Length of the value left in the datastore. See sources for more context.
-    let value_len = 10;
+    let value_len = 4;
```

Comment:
Cannot the SC generate an event, retrieve it in order to avoid this hardcoding?

Reply:
That's a good idea to avoid hardcoding in this case, but I believe it is not the place to make that change

Reply:
Would you mind adding an issue in github and add the link in a comment then?

Reply:
https://github.com/massalabs/massa/issues/3236